i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Airflow 离线同步示例
+sidebar_position:  3
+---
+在下面的内容中，将通过一个完整的示例介绍如何使用 Apache InLong 创建 Airflow 调度任务，并完成 Pulsar -> MySQL 的离线数据同步。
+
+## 环境部署
+### 安装 InLong
+
+在开始之前，我们需要安装 InLong 的全部组件，这里提供两种方式：
+- [Docker 部署](deployment/docker.md)（推荐）
+- [Bare Metal 部署](deployment/bare_metal.md)
+
+### 添加 Connectors
+
+下载与 Flink 版本对应的 [connectors](https://inlong.apache.org/zh-CN/downloads) ，解压后将 `sort-connector-jdbc-[version]-SNAPSHOT.jar` 放在 `/inlong-sort/connectors/` 目录下。
+> 当前 Apache InLong 的离线数据同步能力只支持 Flink-1.18 版本，所以请下载 1.18 版本的 connectors。
+
+## 创建集群和数据目标
+
+### 创建集群标签
+![airflow_create_cluster_labels](img/pulsar_mysql/airflow/airflow_create_cluster_labels.png)
+
+### 注册 Pulsar 集群
+
+![airflow_create_pulsar_cluster](img/pulsar_mysql/airflow/airflow_create_pulsar_cluster.png)
+
+### 创建数据目标
+
+![airflow_create_data_target](img/pulsar_mysql/airflow/airflow_create_data_target.png)
+
+执行如下 Sql 语句：
+
+```mysql
+CREATE TABLE sink_table (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    create_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+```
+
+## Airflow 初始化
+
+### 获取初始 DAG
+
+它们可以在 [Inlong](https://github.com/apache/inlong) 获取。
```

Comment:
What do they refer to? Please describe clearly

Reply:
sure! done.